### PR TITLE
Remaining test server swaggers

### PR DESCRIPTION
**Pull request by MiYanni**
Update the list to accurately represent the remaining test server swaggers to do. 6 remain.

### CODE REVIEW

File: eng/Generate.ps1 [hunk 1]
```diff
@@ -52,15 +52,10 @@ $configurationPath = Join-Path $testServerDirectory 'readme.tests.md'
...
     'additionalProperties',
-    #'azure-composite-swagger',
     #'azure-parameter-grouping',
-    #'azure-report',
-    #'azure-resource',
-    #'azure-resource-x',
     #'azure-special-properties',
     'body-array',
     'body-boolean',
-    #'body-boolean.quirks',
```
Comment (pakrym) [inline eng/Generate.ps1]: What's in all these quirks swaggers?

Comment (MiYanni) [reply eng/Generate.ps1]: Lol.
image

Comment (pakrym) [reply eng/Generate.ps1]: Should we delete them from the test server? Along with other swaggers without tests?

Comment (MiYanni) [reply eng/Generate.ps1]: There might be v2 generators that use them to test things. We have no idea. 🐢

**Pull request merged**